CHANGES

```diff
@@ -1,3 +1,39 @@
+Version 20200107 (phpdvdprofiler.20200107.zip)
+Changes since version 20191229 (phpDVDProfiler.20191229.tar.gz)
```

Comment:
I know the previous changelog had it's own particular format, and I think it is fine keeping all the old entries that way for historic reasons, can we at least switch to the Imperative mood, to match both read better as match the git commit format?

Reply:
Done